packages/preview/touying-unistra-pristine/1.0.0/typst.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "touying-unistra-pristine"
+version = "1.0.0"
+entrypoint = "src/lib.typ"
+authors = [
+  "Enzo Doyen <https://edoyen.com/>"
+]
+license = "MIT"
+description = "Touying theme adhering to the core principles of the style guide of the University of Strasbourg, France"
+# homepage = ""
+repository = "https://github.com/spidersouris/touying-unistra-pristine"
+keywords = ["touying", "unistra", "strasbourg", "presentation"]
+categories = ["presentation"]
+disciplines = []
+compiler = "0.11.0"
+exclude = []
```

Comment:
It would be best to exclude the files that are not needed for the package to work. Unless I'm mistaking that would be the `example` folder and the two cat images.

Reply:
Correct, done